tests/unit/test_calculators.py

```diff
@@ -69,7 +73,23 @@ def test_calc_aligned_file_size(
 def test_calc_tarball_size(
     name_lens: List[int], data_size: int, expected_size: int
 ) -> None:
-    names = ["0" * name_len for name_len in name_lens]
+    try:
+        # Assuming name_lens size is less than 10
+        names = [f"{i}" * name_len for i, name_len in enumerate(name_lens)]
+        cwd = Path(__file__).parent.resolve()
+        test_dir_path = cwd / "test"
+        os.mkdir(test_dir_path)
```

Comment:
Consider using of [TemporaryDirectory](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory)